.pre-commit-config.yaml

```diff
@@ -37,9 +37,9 @@ repos:
     rev: 6.1.0
     hooks:
     -   id: flake8
--   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.5.1
-    hooks:
-    -   id: mypy
-        additional_dependencies: [types-all]
-        exclude: ^testing/resources/
+# -   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Why the comment?

Reply:
It's conflicting with `ruff`, will add that one in a next iteration as a replacement